README.md

```diff
@@ -56,6 +58,8 @@ Vision Tools
 
 In the bitbots_vision_tools directory, special tools for debugging/introspection purposes are provided.
 
+TODO!
```

Comment:
I would propose to open an Issue instead of this.

Reply:
There is already the issue #42 which will be resolved soon by this PR.